tests/flow/test_graph_info.py

```
@@ -461,89 +461,89 @@ def issue_2_query(g, q1, q2):
+       # test flow:
+       # 1. write to replica, expect no telemetry
+       # 2. write to master, expect telemetry on both master & replica
+       # 3. failover
+       # 4. write to new replica, expect no telemetry
+       # 5. write to new master, expect telemetry on both master & replica
+
+       performed_failover = False
+       self.master_graph  = Graph(self.master, "start")
+       self.replica_graph = Graph(self.replica, "start")
+
+       for i in range(2):
```

Ruff (0.15.12)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)